.github/workflows/ci.yml

```diff
@@ -0,0 +1,34 @@
+name: libsecp256k1-ci
+run-name: libsecp256k1 CI
+on: [pull_request, push]
+
+env:
+  SECP256K1_BENCH_ITERS: 2
+  SECP256K1_TEST_ITERS: 16
+
+jobs:
+  win64-native:
+    name: "x86_64: Windows (VS 2022)"
+    # See: https://github.com/actions/runner-images#available-images.
+    runs-on: windows-2022
+
+    strategy:
+      matrix:
+        build_shared_libs: ["ON", "OFF"]
+
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v3
+
+      - name: Generate buildsystem
+        run: cmake -E env CFLAGS="/WX" cmake -B build -A x64 -DSECP256K1_ENABLE_MODULE_RECOVERY=ON -DSECP256K1_BUILD_EXAMPLES=ON -DBUILD_SHARED_LIBS=${{ matrix.build_shared_libs }}
+
+      - name: Build
+        run: cmake --build build --config RelWithDebInfo -- /p:UseMultiToolTask=true /p:CL_MPcount=3
```

Comment:
optional nit: The 3 could be computed as n + 1, where n is read from the system automatically, same below.

Reply:
[Done](https://github.com/bitcoin-core/secp256k1/pull/1389#issuecomment-1670843751l).

In the MSBuild option list, it has been replaced with the high level [`-maxCpuCount`](https://learn.microsoft.com/en-us/visualstudio/msbuild/building-multiple-projects-in-parallel-with-msbuild?view=vs-2022#-maxcpucount-switch) switch.